### DISCUSSION TITLE
How do I review a translation?

### DISCUSSION THREAD

**Discussion by twMat** [TiddlyWiki/TiddlyWiki5#6876] [Q&A]
The tiddler [Translate TiddlyWiki into your language](https://tiddlywiki.com/#Translate%20TiddlyWiki%20into%20your%20language) should also explain how I modify existing translations. It does provide a general link to https://tiddlywiki.com/dev for "...details of creating and maintaining translations" but I don't find it clear what to look for there. I only find [How to create a translation for TiddlyWiki](https://tiddlywiki.com/dev/#How%20to%20create%20a%20translation%20for%20TiddlyWiki) which I don't think explains how/where to *edit* translated tiddlers.

**Comment by pmario** [comment]
There is the translators edition: https://tiddlywiki.com/editions/translators/
You can switch to Swedish language and see all the strings. There are some translated strings and some of them are not translated yet. .... 

The filter dropdown is a bit confusing, since it treats every existing translated string as _not translated_ So you need to change a string, that it counts against the filters. .. You can save the edition locally and send it per ZIP here, or mail to Jeremy ... He will implement it.

**Reply by twMat** [reply]
Thanks.

**Reply by twMat** [reply]
OK, here is a Zip

[swedish.zip](https://github.com/Jermolene/TiddlyWiki5/files/9259144/swedish.zip)
 
[rant]
Ouch, it was more than I expected. So I only corrected a few things. An interesting tidbit was that the previous translator had opted to translate "tiddler title" into (the Swedish equivalent of) "tiddler headline" which is neither correct nor more informative. It is simply not a headline, it *is* the title of the tiddler, and the Swedish word for title is even identical to the English word so it is surprising why someone would deliberately change it. Not to mention the lost alliteration ;)

A problem I'm noting is that there is just so much lingo that really would be *unfortunate* to translate. But I guess that *is* part the difficulty with the task. On the other hand... I'd be curious to know how many actually use the Swedish translation... most Swedes are decent in English and for the TW target group I think the absolute majority prefer English as the system language. There is an irony also; if you are not proficient with lingo like "filter expression", "default alternative", "editor tool", etc then you will likely need more help overall, but you use translated lingo then it will be difficult to formulate the question so others understand what you refer to. (I get this for Windows; I choose Swedish only so that I don't risk getting the wrong keyboard and local settings etc but if I need help on some forum I frustratedly curse Bill Gates...)

**Reply by pmario** [reply]
@Jermolene .. Mat did some translations for the Swedish version

**Comment by pmario** [comment]
The second possibility would be to use git and clone the repository, so you can make changes directly to the files with a text or code editor. 
If you want to try that, I'd be willing to create a howto - video

**Comment by twMat** [comment]
By the way, I don't like the table display with the dropdowns at the side. When making a translation is is important to see the original text without any fuzz. It would be better if the English was displayed immediately above or below the editor field. Is this worth posting a formal issue for?

**Reply by pmario** [reply]
I think that's worth a new issue. ... BUT have a look, if there is an existing one first ;)

**Reply by twMat** [reply]
OK, I didn't find any previous issue post on this so I filed #6879
